app/gen-server/lib/homedb/UsersManager.ts

```diff
@@ -766,4 +788,16 @@ export class UsersManager {
     }
     delta.users = users;
   }
+
+  private _buildExistingUsersByLoginRequest(
+    emails: string[],
+    manager?: EntityManager
+  ) {
+    const normalizedEmails = emails.map(email=> normalizeEmail(email));
+    return (manager || this._connection).createQueryBuilder()
+      .select('user')
+      .from(User, 'user')
+      .leftJoinAndSelect('user.logins', 'logins')
+      .where('email IN (:...emails)', {emails: normalizedEmails});
```

Comment:
@hexaltation is it possible for the list of emails to be empty? We're seeing a failure when running against postgres. Not sure yet if this is the code causing it, might not be, but the error message is of the form you can get with an empty in list:

```
zing=> SELECT 1 WHERE 1 IN ();
ERROR:  syntax error at or near ")"
```

Reply:
Hello again @paulfitz!

After discussion with @hexaltation, we are working on something we hope to fix the issue.

Especially we found a case where `delta.users` is an empty object.

We'll keep you inform about our progress or if we still have questions.

Reply:
Thanks @fflorent, @hexaltation ! I think reasonable behavior to mimic here is SQLite's, where `IN ()` simply matches nothing (and is not a syntax error). This could be done in the code by skipping the query if the where clause will be trying to filter for the the empty set of emails.